data/card_catalogue/unique-1.json

```diff
@@ -264,27 +184,17 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Unarmed attacks deal double damage until you play your next card.",
+      "rules": "Attacks deal double damage until you play your next card.",
       "flavour": "I'm not unarmed, I've got two of them.",
       "image": "assets/core/card/illustration/Bashbolster.png"
     }
   },
-  {
-    "title": "Glass Hammer",
-    "upgrade": "",
-    "tier": "blue",
-    "details": {
-      "rules": "Deal 10 damage to a visible target. Return this card to the deck after casting it.",
-      "flavour": "Bonk",
-      "image": "assets/core/card/illustration/Glass Hammer.png"
-    }
-  },
```

Comment:
I think the "return to deck" mechanic feels a bit off

Reply:
I think this is the right place for a "Cast"/"Active" glyph as we'll be building the discard-after-use mechanic into its definition.

⚡ Deal an additional 10 melee damage

Reply:
Hm, I think the discard that occurs when you play a card is a little different though, because you get it back at the end of the session. With the old wording of "return it to the deck" I don't think you would get it back

Reply:
Regardless, we can re-add this later once we've hammered out how it'll work with the glyphs